### ISSUE TITLE
Restoring from a catastrophe with only database backups and the media folder

### ISSUE THREAD

**Issue by rcrath** [anvc/scalar#178]
Hi, I had a catastrophe and lost my scalar install with all the backups (two local hard disk fails, backup company abruptly stopped supporting linux, errant dean deleted the actual site.   I have reconstructed the database backups from the broken hard drives and have most of the images in the media folder.  I am after one book as a test case and imported that book's book ID, Contents, user (just me) and user books.  I have gotten things to line up and show up in scalar as far as the book appearing in the dashboard, with me as the author.  Although the "foreign" bookID in the content table and the "referent" book id  in the books table are the same, I have been unable to get the content to show. Would this be possible to repair?  This particular project has been worked on by dozens of student and myself for years and was just about ready to release when it got deleted.

the site is at http://udigix.net/scalar2/souls-of-black-folk/index

To recap, the book title is read, along with the user attached to the book, but no pages are seen nor media. The page content is in the database and the media is in the /media folder.

LMK is you need more info on the install or anything.  It is an uptodate fresh install of scalar on reclaim hosting if that helps.

**Comment by rcrath** [comment]
My guess is that for some reason, the database connection fails for the content and media but not for book ID or User (or Book Users). It might be a scalar version thing since this was from a 2019 version of scalar.  Or maybe a PHP 7 problem?
